packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_model.dart

```diff
@@ -97,11 +104,12 @@ class LoggingTableModel extends ChangeNotifier {
   }
 
   /// Get the height of a filtered Log at [index].
-  double getFilteredLogHeight(int index) {
+  double getFilteredLogHeight(BuildContext context, int index) {
     final cachedHeight = cachedHeights[index];
     if (cachedHeight != null) return cachedHeight;
 
     return cachedHeights[index] = LoggingTableRow.calculateRowHeight(
+      context,
```

Comment:
we should not be passing BuildContext around. Just rebuild the text style with the defaultFontSize instead of looking up the theme.